incubator/nodejs-loopback/templates/scaffold/.appsody-config.yaml

```diff
@@ -0,0 +1 @@
+stack: appsody/nodejs-loopback:0.1.0
```

Comment:
could you remove the `.0` on the end so that it always picks up the latest:
eg: 
> stack: appsody/nodejs-loopback:0.1